### PR TITLE
fix: update merge spell action to bump version.xcconfig SQPIT-773

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-773" title="SQPIT-773" target="_blank">TaskSQPIT-773</a>  update Github action merge PR to update version.xcconfig
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Recent framework release does not update `version.xcconfig`

### Causes (Optional)

It was done by Jenkins script, after we use `mergePR` action this feature is not included

### Solutions

Add this feature to mergePR action.
This action is tested in https://github.com/wireapp/wire-ios-canvas/pull/33
Add @agisilaos and @sb88k to admin list so that they can use this spell

### CODE REVIEW

**Review by johnxnguyen — APPROVED** [review]

File: .github/workflows/mergePR.yml [hunk 1]
```diff
@@ -41,7 +43,48 @@ jobs:
...
+          # process version.xcconfig
+          curl --silent https://raw.githubusercontent.com/wireapp/wire-ios-shared-resources/master/Scripts/updateVersionXcconfig.swift --output updateVersionXcconfig.swift
+          chmod +x updateVersionXcconfig.swift
+          swift updateVersionXcconfig.swift ./Resources/Configurations/version.xcconfig ${{ steps.tag_version.outputs.new_tag }}
+          rm updateVersionXcconfig.swift
+
+          git config --global user.email "iosx@wire.com"
+          git config --global user.name "zenkins"
+          git add .
+          git commit -m"update version.xcconfig for '${{ steps.tag_version.outputs.new_tag }}'" || true
```
Comment (typfel) [inline .github/workflows/mergePR.yml:84]:
```suggestion
          git commit -m"chore: Update version.xcconfig for '${{ steps.tag_version.outputs.new_tag }}'" || true
```

**Review by typfel — CHANGES_REQUESTED** [review]
Let's make the commit message adhere to semantic commit message: https://sparkbox.com/foundry/semantic_commit_messages

**Comment by typfel** [comment]
So are currently copying this script to all repos? Would it make sense to move it to the global `.github` folder? https://github.com/wireapp/.github ?

**Comment by billypchan** [comment]
@typfel I will try to move this workflow to organisation level. Ref: https://docs.github.com/en/actions/learn-github-actions/reusing-workflows
We can merge this PR first and then create a new PR for moving workflows

**Review by typfel — APPROVED** [review]

**Pull request merged**